### DISCUSSION TITLE
can we upload other fonts that aren't supported currently?

### DISCUSSION THREAD

**Discussion by Amirko28** [ONLYOFFICE/DocumentServer#1904] [Q&A] [answered]
there are several fonts that lose their design when uploaded to OnlyOffice and we were wondering if it's possible for us to add new fonts to the system.

**Comment by ShockwaveNN** [comment, marked as answer]
This may help you

https://helpcenter.onlyoffice.com/installation/docs-community-install-fonts-linux.aspx
